content/tracing/setup/environment.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Environment
+kind: documentation
+further_reading:
+- link: "/tracing/setup/docker"
+  tag: "Documentation"
+  text: Docker setup
+- link: "/tracing/setup/go"
+  tag: "Documentation"
+  text: Go language instrumentation
+- link: "/tracing/setup/java"
+  tag: "Documentation"
+  text: Java language instrumentation
+- link: "/tracing/setup/python"
+  tag: "Documentation"
+  text: Python language instrumentation
+- link: "/tracing/setup/ruby"
+  tag: "Documentation"
+  text: Ruby language instrumentation
+---
+
+#### Definition
+
+An environment is a first class dimension that you can use to scope an entire Datadog APM application. Some display settings can be shared across environments, but all the measurable data (traces/metrics/statistics) can not be re-aggregated across multiple environments. Use cases can be:
```

Comment:
`Use cases can be:` => `Use cases include:`

Reply:
👌